//#include <iostream>
//#include <exception>
//#include <vector>
//#include <string>
//using namespace std;
//int main()
//{
//	vector< int > v;
//	v.push_back(1000);
//	try
//	{
//		v.at(2) = 123;
//	}
//	catch(exception &ex)
//	{
//		cout<<ex.what() << endl;
//	}
//	catch(...)
//	{
//		cout<< "....\n";
//	}
//
//
//	string line = "a 123";
//	try
//	{
//		stol(line);
//	}
//	catch(invalid_argument & ex)
//	{
//		cout << ex.what() << endl;
//	}
//	
//}